frontend/src/app/offers/create-offer/create-offer.component.ts

```diff
@@ -0,0 +1,80 @@
+import { Component, OnInit } from '@angular/core';
+import { Offer } from 'app/homepage-offer/offers.model';
+import { AuthService } from 'app/auth.service';
+import { User } from 'app/user';
+import { Observable } from 'rxjs/Observable';
+import 'rxjs/add/operator/map';
+import { OffersService } from 'app/homepage-offer/offers.service';
+import { ActivatedRoute, Params } from '@angular/router';
+import { environment } from 'environments/environment';
+
+@Component({
+  selector: 'volontulo-create-offer',
+  templateUrl: './create-offer.component.html',
+  styleUrls: ['./create-offer.component.scss']
+})
+export class CreateOfferComponent implements OnInit {
+  djangoRoot: string;
+  public inEditMode: boolean = false;
+  offer: Offer = new Offer;
+  user: User;
+  isAdmin: boolean = false;
+  hasOrganization = false;
+
+  constructor(
+    private authService: AuthService,
+    private offersService: OffersService,
+    private route: ActivatedRoute,
+  ) {
+      this.djangoRoot = environment.djangoRoot;
+  }
+
+  ngOnInit() {
+    this.authService.user$
+    .subscribe(
+      response => {
+        this.user = response;
+        if(this.user){
+          this.isAdmin = this.user["isAdministrator"];
+          this.hasOrganization = this.user["organizations"].length > 0;
+        }
+      }
+    );
+
+    this.route.params
```

Comment:
I think you could use some rxjs power here (but it will be a bit more difficult). Overall, nested subscriptions quite often (usually?) indicate that there's something wrong with the code and it might be written more "rxjs way".

I.e. something (just a simplified example here) in the direction of:
```
this.offer$ = this.route.params
  .map(params => params.offerId)
  .switchMap(offerId => this.offersService.getOffer(offerId))
```

Reply:
@radoslawroszkowiak please, take a look below if this is what you meant by rxjs power and if it's properly done.